src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -253,14 +194,16 @@ class DetailsCard extends React.Component {
     }
 
     if (prevProps.storageDomains !== this.props.storageDomains) {
-      this.setState({ isoList: createIsoList(this.props.storageDomains) }) // eslint-disable-line react/no-did-update-set-state
+      const vmClusterId = this.props.vm.getIn(['cluster', 'id'])
```

Comment:
I think it would better to move it outside condition, because cluster id is used a few times in this function.

Reply:
Done.